### DISCUSSION TITLE
Compare two date fields

### DISCUSSION THREAD

**Discussion by Frozzesav** [Megabit/Blazorise#2895] [Q&A] [answered]
Hello! I'm not good  in programming yet.
Help pls. 

How to compare two date fields inserting row in datagrid?
One of date should greater than second. And I want to show ErrorText if date less.

Thanks)

**Comment by stsrki** [comment, marked as answer]
It should be possible with data-annotations and by creating a custom validator attribute. eg. https://stackoverflow.com/questions/17733430/compare-two-properties-with-custom-dataannotation-attribute

**Reply by Frozzesav** [reply]
Thank you a lot!
